src/components/EncounterDetails/EncounterDetails.jsx

```diff
@@ -1,5 +1,165 @@
 import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
 import './EncounterDetails.css';
 
+export default function EncounterDetails({ encounters }) {
+  const [selectedMonsters, setSelectedMonsters] = useState([]);
+
+  useEffect(() => {     
+    console.log(encounters, "before if")
+    if (encounters && encounters[0]) {
```

Comment:
Right now, we have everything hard coded to the first encounter. We'll need to adjust this to map to the encounter that's preview was clicked. I think it's a-ok to do that in a separate ticket and PR. 👍

Reply:
Works for me!